src/sdk/pynni/nni/nas/pytorch/fixed.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+import torch
+
+from nni.nas.pytorch.mutables import MutableScope
+from nni.nas.pytorch.mutator import Mutator
+
+
+class FixedArchitecture(Mutator):
+
+    def __init__(self, model, fixed_arc, strict=True):
+        """
+        Initialize a fixed architecture mutator.
+
+        Parameters
+        ----------
+        model: nn.Module
```

Comment:
`model:` --> `model :`

Reply:
My IDE told me there is no space. I think I'm in big trouble now.